frontend/src/features/customer/components/tabs/HelperTab.tsx

```diff
@@ -1,23 +1,63 @@
-import { Tabs } from '@components';
+import { Spinner, Tabs } from '@components';
 import { HelperCard, HelperEmptyCard, HelperSelectInfoCard } from '@customer/components';
+import { useParams, useNavigate } from '@tanstack/react-router';
+import { getApplicationListById } from '@customer/apis';
+
+const HelperTab = ({ status }: { status: string }) => {
+  const navigate = useNavigate();
+  const { escortId } = useParams({ from: '/customer/escort/$escortId/' });
+  const { data: applicationList, isLoading } = getApplicationListById(Number(escortId));
+
+  const handleHelperCardClick = (helperId: number, escortId: string, applicationId: number) => {
+    if (status === '매칭중') {
+      navigate({
+        to: '/customer/escort/$escortId/$helperId/helper/$applicationId',
+        params: {
+          escortId: escortId,
+          helperId: helperId.toString(),
+          applicationId: applicationId.toString(),
+        },
+        search: { canSelect: 'true' },
+      });
+    } else {
+      navigate({
+        to: '/customer/escort/$escortId/$helperId/helper/$applicationId',
+        params: {
+          escortId: escortId,
+          helperId: helperId.toString(),
+          applicationId: applicationId.toString(),
+        },
+        search: { canSelect: 'false' },
+      });
+    }
+  };
```

Comment:
```suggestion
  const handleHelperCardClick = (helperId: number, escortId: string, applicationId: number) => {
    navigate({
    to: '/customer/escort/$escortId/$helperId/helper/$applicationId',
    params: {
        escortId: escortId,
        helperId: helperId.toString(),
        applicationId: applicationId.toString(),
    },
    search: { canSelect: status === '매칭중' ? 'true' : 'false' },
    });
  };
```

이런식으로는 처리가 안되나요?

추가로 저도 true/false string을 선호하지는 않는데,
searchParam으로 status=matching 이런식으로 들어가는건 어떨까요?